psci/Parser.hs

```diff
@@ -86,3 +85,6 @@ parseCommand cmdString =
     where
     manyDecls :: P.TokenParser [P.Declaration]
     manyDecls = C.mark (many1 (C.same *> P.parseDeclaration))
```

Comment:
Regarding the `let import ...` issue, I think we can fix that by using `parseLocalDeclaration` here instead of `parseDeclaration`.

If we were going to do that, though, I think we should allow bare data, type synonym, type class, and type instance declarations (ie, without a 'let') at the same time, so that we aren't removing useful features that people might be using.